IOSExercises/Views/ColorsTableViewController.h

```diff
@@ -19,9 +19,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ColorsTableViewController : UITableViewController
 
-@property (nonatomic, weak) id<ColorDelegate> delegate;
+@property(nonatomic, weak) id<ColorDelegate> delegate;
```

Comment:
```suggestion
@property (nonatomic, weak) id<ColorDelegate> delegate;
```

Spacing. Please apply to all